pkg/engine/engine.go

```diff
@@ -744,10 +744,7 @@ func getClusterAutoscalerAddonFuncMap(addon api.KubernetesAddon, cs *api.Contain
 			return api.GetClusterAutoscalerNodesConfig(addon, cs)
 		},
 		"GetVMType": func() string {
-			if cs.Properties.AnyAgentUsesVirtualMachineScaleSets() {
-				return base64.StdEncoding.EncodeToString([]byte("vmss"))
-			}
-			return base64.StdEncoding.EncodeToString([]byte("standard"))
```

Comment:
@jackfrancis do you have any idea why we are base64 encoding the result here?

Reply:
looks like it's being used as data in addon yaml specs, I renamed the function so it's clear

Reply:
Right. We should rename this to something like `GetVMTypeForClusterAutoscaler`

Reply:
`GetBase64EncodedVMType` also works :)